tests/hipscat/io/test_paths.py

```diff
@@ -71,3 +74,12 @@ def test_pixel_association_directory_nonint():
     """Simple case with non-integer inputs"""
     with pytest.raises(ValueError):
         paths.pixel_association_directory("/foo", "zero", "five", "zero", "four")
+
+
+def test_is_valid_catalog(tmp_path, small_sky_catalog, small_sky_pixels):
+    """Tests existence of the catalog_info and partition_info files"""
+    catalog_dir_fp = get_file_pointer_from_path(tmp_path)
+    assert not is_valid_catalog(catalog_dir_fp)
+    write_catalog_info(catalog_dir_fp, small_sky_catalog.catalog_info)
```

Comment:
Could you add another assert in here to make sure it's not valid with just one of the files?